accord-core/src/main/java/accord/utils/Invariants.java

```diff
@@ -27,13 +27,16 @@
 
 public class Invariants
 {
-    // TODO (now): configure by system parameter and turn off by default
-    private static final boolean PARANOID = true;
-    private static final boolean DEBUG = true;
+    private static final int PARANOIA = Integer.parseInt(System.getProperty("accord.paranoia", "0"));
+    private static final boolean DEBUG = System.getProperty("accord.debug", "false").equals("true");
```

Comment:
does this compile in C*?  We have a check style (that does check Accord code) that we don't touch these APIs

Reply:
It compiles. We don’t have the same API available to accord. We could introduce a parallel one for Accord